spec/log_parser_spec.rb

```diff
@@ -6,7 +6,14 @@
   end
 
   it 'must have the text' do 
-    data = LogParser.new('test_file.log')
+    data = LogParser.new('./spec/fixtures/test_file.log')
     expect(data.read_first_line).to eq("  0:00 ------------------------------------------------------------\n")
   end
 end
+
+describe '#json_output' do
+  it 'should return the number of lines in a json object' do
```

Comment:
Por convenção no rspec não é recomendado utilizar o `should` pois as mensagens de teste devem ser assertivas.

https://www.betterspecs.org/#should

Reply:
modifiquei